src/utils/escape.test.ts

```diff
@@ -38,4 +39,18 @@ describe("escaping", () => {
             expect(Cypher.utils[func](original)).toBe(expected);
         });
     });
+
+    describe("escapeLiteralString", () => {
+        test("Does not escape empty strings", () => {
+            expect(escapeLiteralString("")).toBe("");
+        });
+
+        test.each([[`my "var"`, `my \\"var\\"`, `my \\"var`, `my \\\\"var`]])("Escape '%s'", (original, expected) => {
```

Comment:
I believe there is a typo here: there is only one element in the list.

Reply:
good catch